mscore/revision.h

```diff
@@ -1 +1 @@
-3543170
+a504e39
```

Comment:
I don't know how this got included, .gitignore should ignore revision.h. Nevertheless, this file should not be in the commited files.

Reply:
removed